burrow/check.py

```diff
@@ -0,0 +1,86 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# 3rd party
+import requests
+
+# project
+from checks import AgentCheck
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'burrow'
+
+
+class Burrow(AgentCheck):
+
+    LIST_CLUSTERS = '/v2/kafka'
+    CLUSTER_DETAILS = '/v2/kafka/{cluster}'
+    LIST_CONSUMERS = '/v2/kafka/{cluster}/consumer'
+    CONSUMER_LAG = '/v2/kafka/{cluster}/consumer/{consumer}/lag'
+
+    @staticmethod
+    def _check_for_error(req):
+        if req.status_code != 200:
+            raise Exception("Got {} error from burrow http endpoint".format(req.status_code))
+
+        res = req.json()
+        if res['error']:
```

Comment:
is the `error` field always present in the json object? If it's not you might want to use `if res.get('error'):` instead